src/modules/ekf2/EKF/aux_global_position.cpp

```diff
@@ -119,6 +122,13 @@ void AuxGlobalPosition::update(Ekf &ekf, const estimator::imuSample &imu_delayed
 			if (continuing_conditions) {
 				ekf.fuseHorizontalPosition(aid_src);
 
+				if (isTimedOut(aid_src.time_last_fuse, imu_delayed.time_us, ekf._params.no_aid_timeout_max)
```

Comment:
I'm a bit unclear about `reset_timeout_max` vs `no_aid_timeout_max` here?


```suggestion
				if (isTimedOut(aid_src.time_last_fuse, imu_delayed.time_us, ekf._params.reset_timeout_max)
```

https://github.com/PX4/PX4-Autopilot/blob/ff4d8d0a884deb58a1b39bbc1f0468ac7e8a443f/src/modules/ekf2/EKF/common.h#L472-L473

Reply:
Let me ask @priseborough about that. It's also not really clear to me and the usage in the code looks a bit arbitrary